artifactory/services/go/publishwithmatrixparams.go

```diff
@@ -20,12 +20,11 @@ type publishWithMatrixParams struct {
 func (pwmp *publishWithMatrixParams) isCompatible(artifactoryVersion string) bool {
```

Comment:
I suggest adding a test function, which will test this function with a few Artifactory versions.

Reply:
Its being tested in the go_test.go file. There is a test function to check the compatible publisher. The compatible publisher checks the isCompatible of each one that using the interface. I think it's already covered there.